osf/migrations/0030_preprintprovider_domain.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hurray for migrations!!

One request: typically when a PR calls for a migration, we call it out very explicitly as a separate PR section, and arrange for the migration to be tracked in the go/nogo ticket. Basically, the goal is to 100% ensure we take care of it. 

It's possible that policy has changed since I last checked, but repo maintainer can advise on current preference.